dt-users/user-initial-setup.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/**
+ * Show a modal prompting the user with the following:
+ *  - Default language to be adopted following initial login.
+ */
+
+if ( !defined( 'ABSPATH' ) ){
+    exit; // Exit if accessed directly
+}
+
+function process_user_initial_setup_language_updates(){
```

Comment:
@kodinkat  lets use the  `update_user_contact_info()`  function and add the `dt_user_initial_setup_default_language` to it. 
This way if the user sets their language in the mobile app, or a different way, then they won't get asked again.